.github/workflows/release-open-issue-warning.yml

```
@@ -0,0 +1,169 @@
+            console.log(`Searching for open issues and PRs in milestone: ${milestone}`);
+            
+            try {
+              // Search for issues in the milestone
+              const searchQuery = `milestone:"${milestone}" state:open repo:${context.repo.owner}/${context.repo.repo}`;
+            
+              const searchResults = await github.rest.search.issuesAndPullRequests({
+                q: searchQuery,
+                per_page: 20,
+                advanced_search: true
+              });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)